lib/std/os.zig

```diff
@@ -6056,30 +6056,39 @@ pub fn sendto(
             .SUCCESS => return @as(usize, @intCast(rc)),
 
             .ACCES => return error.AccessDenied,
+            .ADDRINUSE => return error.AddressInUse,
+            .AFNOSUPPORT => return error.AddressFamilyNotSupported,
             .AGAIN => return error.WouldBlock,
             .ALREADY => return error.FastOpenAlreadyInProgress,
-            .BADF => unreachable, // always a race condition
+            .BADF => unreachable, // maybe a race condition. (AF_UNIX) the sent file descriptor is not open.
```

Comment:
Why (AF_UNIX) here? man page sayys `sockfd` is not a valid file descriptor.

Reply:
in `man 7 unix`